src/Calculator/Controls/EquationTextBox.cpp

```diff
@@ -144,7 +144,7 @@ void EquationTextBox::OnRichEditBoxTextChanged(Object ^ sender, RoutedEventArgs
 
 void EquationTextBox::OnRichEditBoxGotFocus(Object ^ sender, RoutedEventArgs ^ e)
 {
-    m_isFocused = true;
+    m_HasFocus = true;
```

Comment:
Is this only for RichEditFocus of for the whole control?

Reply:
Technically the whole control, but currently we only care when the RichEditBox gets focus